net/src/client/tcp/actors/jsonrpc.rs

```diff
@@ -82,12 +127,30 @@ impl JsonRpcClient {
 
 impl Actor for JsonRpcClient {
     type Context = Context<Self>;
+
+    fn started(&mut self, _ctx: &mut Self::Context) {
+        log::debug!("JsonRpcClient actor started!");
+    }
+
+    fn stopping(&mut self, _ctx: &mut Self::Context) -> Running {
+        log::info!("JsonRpcClient actor was trying to stop for some reason. Refusing to stop!");
+
+        Running::Continue
+    }
+
+    fn stopped(&mut self, _ctx: &mut Self::Context) {
+        log::info!("JsonRpcClient actor stopped!");
+    }
 }
 
 impl Supervised for JsonRpcClient {}
 
-/// TODO: doc
-pub struct Notification(pub Value);
+/// JSONRPC notification.
+///
+/// Contains:
+/// 1. Subscription ID
+/// 2. A JSON value
+pub struct Notification(pub String, pub Value);
```

Comment:
There is no need to keep unnamed fields

```suggestion
/// JSONRPC notification.
pub struct Notification {
    /// Subscription ID
    pub subscription_id: String,
    /// A JSON value
    pub value: Value,
}
```